<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../images/mantid_imaging_64px.png</normaloff>../../../images/mantid_imaging_64px.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* background: url(/home/matt/Documents/imgpy/mantidimaging/gui/ui/code.jpg)*/</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadDataset"/>
    <addaction name="actionLoadImages"/>
    <addaction name="actionLoadNeXusFile"/>
    <addaction name="actionSampleLoadLog"/>
    <addaction name="actionLoadProjectionAngles"/>
    <addaction name="actionLoad180deg"/>
    <addaction name="separator"/>
    <addaction name="actionSaveImages"/>
    <addaction name="actionSaveNeXusFile"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWorkflow">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Workflow</string>
    </property>
    <addaction name="actionFilters"/>
    <addaction name="actionRecon"/>
    <addaction name="actionCompareImages"/>
    <addaction name="actionSpectrumViewer"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOnlineDocumentation"/>
    <addaction name="actionAbout"/>
    <addaction name="actionWizard"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Image</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWorkflow"/>
   <addaction name="menuImage"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLoadDataset">
   <property name="text">
    <string>Load dataset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveImages">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save as Image Files</string>
   </property>
   <property name="toolTip">
    <string>Save as Image Files (tiff, fit, etc)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionTomopy">
   <property name="text">
    <string>Tomopy</string>
   </property>
  </action>
  <action name="actionRecon">
   <property name="text">
    <string>Reconstruct</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionManual_Center_of_Rotation">
   <property name="text">
    <string>Manual Center of Rotation</string>
   </property>
  </action>
  <action name="actionFilters">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Operations</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionOnlineDocumentation">
   <property name="text">
    <string>Online Documentation</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionTomopyRecon">
   <property name="text">
    <string>TomoPy Reconstruction</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Mantid Imaging</string>
   </property>
  </action>
  <action name="actionSavuFilters">
   <property name="text">
    <string>Savu GUI</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionCompare_Images">
   <property name="text">
    <string>Compare Images</string>
   </property>
  </action>
  <action name="actionCompareImages">
   <property name="text">
    <string>Compare Images</string>
   </property>
  </action>
  <action name="actionSampleLoadLog">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load log for stack...</string>
   </property>
  </action>
  <action name="actionLoad180deg">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load 180 deg projection...</string>
   </property>
  </action>
  <action name="actionLoadProjectionAngles">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load projection angles...</string>
   </property>
  </action>
  <action name="actionLoadImages">
   <property name="text">
    <string>Load images</string>
   </property>
  </action>
  <action name="actionWizard">
   <property name="text">
    <string>Wizard</string>
   </property>
  </action>
  <action name="actionLoadNeXusFile">
   <property name="text">
    <string>Load NeXus File</string>
   </property>
  </action>
  <action name="actionSaveNeXusFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save as NeXus File</string>
   </property>
  </action>
  <action name="actionSpectrumViewer">
   <property name="text">
    <string>Spectrum Viewer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
